packages/esm-patient-common-lib/src/offline/patient.ts

```diff
@@ -1,60 +0,0 @@
-import { usePatient, getSynchronizationItems } from '@openmrs/esm-framework';
-import useSWR from 'swr';
-
-export const patientRegistrationSyncType = 'patient-registration';
```

Comment:
Deleting this constant may have been a mistake, but we don't need the rest of this file...